src/lib/stats/distributions.ml

```diff
@@ -104,3 +104,10 @@ let ln_dirichlet_pdf ~alphas =
 let dirichlet_pdf ~alphas =
   let ln_pdf = ln_dirichlet_pdf ~alphas in
   fun arr -> exp (ln_pdf arr)
+
+
+let benford_pdf ?(base=10) ~digits =
+  if base < 2 then invalidArg "base must be positive at least 2: %d" base else
```

Comment:
oops, that was two half sentences mixed together by accident.  Yes, your wording is better and should be changed accordingly.